frontend/src/routes/(authenticated)/project/[project_code]/SendReceiveUrlField.svelte

```diff
@@ -31,7 +31,7 @@
       <input value={projectHgUrl} class="input input-bordered join-item w-full focus:input-success" readonly />
       <div class="join-item tooltip-open" class:tooltip={copiedToClipboard} data-tip={$t('clipboard.copied')}>
         {#if copiedToClipboard}
-          <IconButton disabled icon="i-mdi-check" style="btn-outline btn-success" />
+          <IconButton fake icon="i-mdi-check" style="btn-outline btn-success" />
```

Comment:
does fake just mean this is just an icon?

Reply:
Yeah 😕. We just happen to have this scenario where it's desireable to replace a real button with a fake button that looks the same, doesn't have ugly "disabled styling", but doesn't actually do anything.